File: Mundo 3/ex108/moeda.py
def dobro(preco):
    res = preco * 2
    return res


def metade(preco):
    res = preco / 2
    return res


def aumenta(preco, porc):
    res = preco + (preco * porc / 100)
    return res


def diminui(preco, porc):
    res = preco - (preco * porc / 100)
    return res


def moeda(preco, moeda="R$"):
    return f"{moeda}{preco}".replace(".", ",")

File: z-ex109/moeda.php
<?php 
class Moeda {
  public function aumentar($n, $taxa, $formato = false){
    $res = $n + ($n * $taxa / 100);
    return $formato ? $this->moeda($res) : $res;
  }
  
  public function diminuir($n, $taxa, $formato = false){
    $res = $n - ($n * $taxa / 100);
    return $formato ? $this->moeda($res) : $res;
  }
  
  public function dobro($n, $formato = false){
    $res = $n * 2;
    return $formato ? $this->moeda($res) : $res;
  }
  
  public function metade($n, $formato = false){
    $res = $n / 2;
    return $formato ? $this->moeda($res) : $res;
  }

  public function moeda($preco, $moeda="R$") {
    return "{$moeda} " . number_format($preco, 2, ",", ".");
  }
}

    